ibis_bigquery/client.py

```diff
@@ -18,14 +18,10 @@
 import ibis.expr.schema as sch
 import ibis.expr.types as ir
 import pandas as pd
-import regex as re
 from google.api_core.client_info import ClientInfo
-from google.api_core.exceptions import NotFound
-from ibis.client import Database, Query, SQLClient
+from ibis.client import Database, Query
```

Comment:
I get

```
ibis_bigquery/client.py:22: in <module>
    from ibis.client import Database, Query
E   ModuleNotFoundError: No module named 'ibis.client'
```

when trying to run the tests locally against master. I don't see a `client` module here: https://github.com/ibis-project/ibis/tree/master/ibis Did this move?

Reply:
Looks like yes in https://github.com/ibis-project/ibis/pull/3035

Reply:
But the Query class was removed? The only references I find to it now are here: https://github.com/ibis-project/ibis/blob/6f6f0ff77899afbce694a0d25cf3edd0d297f33c/docs/source/user_guide/design.rst

Reply:
`ibis.client` doesn't exist, and it was moved to `ibis.backends.base`. `Query` was indeed removed, and whatever was in it was moved to `Client`, and later to `Backend`: #2789

For some backends, there is still a `client.py`, that contains `Database` and other things, but the `Client` classes don't exist and everything there have been moved to `Backend`.